scss/_variables.scss

```diff
@@ -301,15 +301,13 @@ $headings-font-weight:        500 !default;
 $headings-line-height:        1.2 !default;
 $headings-color:              null !default;
 
-$display1-size:               6rem !default;
-$display2-size:               5.5rem !default;
-$display3-size:               4.5rem !default;
-$display4-size:               3.5rem !default;
-
-$display1-weight:             300 !default;
-$display2-weight:             300 !default;
-$display3-weight:             300 !default;
-$display4-weight:             300 !default;
+//display types as font-size and font-weight
+$displays: (
+  1:(6rem, 300),
```

Comment:
Let's use nested maps with explicit key-value pairs instead of this setup that requires `nth` value navigation.

Reply:
Now using nested maps with explicit key-value pairs